examples/length/length.bal

```diff
@@ -45,21 +37,28 @@ public function main() {
     length = myStr.length();
     io:println("String size: ", length);
 
+    // This creates a XML element.
```

Comment:
```suggestion
    // This creates an XML element.
```

Reply:
Also wondering if we should remove "This" in all the comments and change them to 
```
// Create an integer array with several integer elements.
// Create an XML element.
```
etc.

Maybe we can do this separately.